src/components/Math.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Math() {
+  return (
+    <div>This is my math</div>
+  );
+}
```

Comment:
- [ ] I will suggest you implement the changes required by the previous reviewer. Change the name of the `Math.js` to `Calculator.js`
- [ ] Secondly, you are not required to use a functional component yet. It will be good to revert back to class based component.

See example:
```js
class Calculator extends Component {
  render() {
    // reference as a js object
    return {
      <div>
       // Calculator app design goes here
      </div>
    };
  }
}
```

Reply:
This issue is well resolved! ✔️